### PR TITLE
Clean up .Legacy

### PR DESCRIPTION
**Pull request by rynowak**
Also sealed two types that didn't get the treatment.

### CODE REVIEW

File: src/Microsoft.AspNetCore.Razor.Language/HtmlAttributeValueStyle.cs [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-namespace Microsoft.AspNetCore.Razor.Language.Legacy
+namespace Microsoft.AspNetCore.Razor.Language
 {
+    // This is the design time equivalent of Microsoft.AspNetCore.Razor.TagHelpers.HtmlAttributeValueStyle.
+    // They should be kept in sync.
     public enum HtmlAttributeValueStyle
```
Comment (rynowak) [inline src/Microsoft.AspNetCore.Razor.Language/HtmlAttributeValueStyle.cs]: @NTaylorMullen - I want to find a better name for this if possible.

Comment (rynowak) [inline src/Microsoft.AspNetCore.Razor.Language/HtmlAttributeValueStyle.cs]: Idea that I had was `AttributeStructure` to go along with `TagStructure`

Comment (NTaylorMullen) [inline src/Microsoft.AspNetCore.Razor.Language/HtmlAttributeValueStyle.cs]: Do it.

File: src/Microsoft.AspNetCore.Razor.Language/Legacy/BlockKindInternal.cs [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 namespace Microsoft.AspNetCore.Razor.Language.Legacy
 {
-    public enum BlockKindInternal
+    internal enum BlockKindInternal
```
Comment (NTaylorMullen) [inline src/Microsoft.AspNetCore.Razor.Language/Legacy/BlockKindInternal.cs]: 😆

**Pull request merged**